apps/web-giddh/src/app/add-company/add-company.component.ts

```diff
@@ -971,9 +982,11 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
             return;
         }
 
-        if (this.selectedStep === 2 && this.thirdStepForm.invalid) {
-            this.isFormSubmitted = true;
-            return;
+        if (this.isNewUserLoggedIn) {
```

Comment:
Can we write like this ??

 if (this.isNewUserLoggedIn && this.selectedStep === 2 && this.thirdStepForm.invalid) {
            this.isFormSubmitted = true;
            return;
      }